frontend/src/components/assessments/TestSubmissionMessage.tsx

```diff
@@ -1,17 +1,23 @@
-import React from "react";
+import React, { useContext } from "react";
 import { Center } from "@chakra-ui/react";
 
 import TestSubmissionIllustration from "../../assets/illustrations/test-submission.svg";
-import { HOME_PAGE } from "../../constants/Routes";
+import AuthContext from "../../contexts/AuthContext";
 import MessageContainer from "../common/MessageContainer";
 
 const TestSubmissionMessage = (): React.ReactElement => {
+  const { setAuthenticatedUser } = useContext(AuthContext);
+
+  const handleClick = () => {
+    setAuthenticatedUser(null);
```

Comment:
Also wondering, I noticed that if you're logged in as a student and you refresh the page, it not only loses your test progress but also logs you out (sends you to the homepage). That behaviour seems a little annoying since they'd have to re-enter their name and code again. Any idea if there's a quick fix?

Reply:
yeah +1. i think carissa & i talked about this a bit before & we wanted to persist login + answers to local storage as a solution to this, so theres a ticket in the backlog for that.

for now, maybe i can add a prompt that shows on reload to warn them from leaving the page? i had it as a TODO for this PR, but forgot to do that

Reply:
yeah whatever you think is best! it's not really related to this PR, I just wanted to mention it so if it's tracked we're good 🟢